package.json

```diff
@@ -40,10 +40,22 @@
   ],
   "moziot": {
     "api": {
-      "min": 1,
+      "min": 2,
       "max": 2
     },
+    "enabled": true,
     "plugin": true,
-    "exec": "{nodeLoader} {path}"
+    "exec": "{nodeLoader} {path}",
+    "config": {
+      "usernames": {}
+    },
+    "schema": {
+      "type": "object",
+      "properties": {
+        "usernames": {
```

Comment:
The code looks fine, but I'm not clear on what this object is supposed to look like. Should usernames actually be an array of objects, like:

```json
[
  {
    "id": "id1",
    "username": "username1"
  }
]
```

Or is it something totally different?

Reply:
It's in the form `{"id1": "username1"}`

Reply:
Ah. If you did it the other way, you could specify the schema better so the user knows what to configure in the UI.

Reply:
Actually, I don't even know if this is intended to be user facing at all. Just ignore me, if not.

Reply:
Yeah, to some extent I don't expect any user to want to pop open the hood here but there's no reason not to. I'll just rewrite all the shiny easy map-based code with the necessary linear searches

Reply:
It'll probably end up being close to the ONVIF adapter's schema: https://github.com/mozilla-iot/onvif-adapter/blob/master/package.json#L54-L89